plugins/rebalance/init.go

```diff
@@ -0,0 +1,53 @@
+package rebalance
+
+import (
+	"github.com/gluster/glusterd2/glusterd2/servers/rest/route"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	"github.com/gluster/glusterd2/pkg/sunrpc"
+)
+
+// Plugin is a structure which implements GlusterdPlugin interface
+type Plugin struct {
+}
+
+// Name returns name of plugin
+func (p *Plugin) Name() string {
+	return "rebalance"
+}
+
+// SunRPCProgram returns sunrpc program to register with Glusterd
+func (p *Plugin) SunRPCProgram() sunrpc.Program {
+	return nil
+}
+
+// RestRoutes returns list of REST API routes to register with Glusterd
+func (p *Plugin) RestRoutes() route.Routes {
+	return route.Routes{
+		route.Route{
+			Name:        "RebalanceStart",
+			Method:      "POST",
+			Pattern:     "/volumes/{volname}/rebalance/start",
+			Version:     1,
+			HandlerFunc: rebalanceStart},
+		route.Route{
+			Name:        "RebalanceStop",
+			Method:      "POST",
+			Pattern:     "/volumes/{volname}/rebalance/stop",
+			Version:     1,
+			HandlerFunc: rebalanceStop},
+		route.Route{
+			Name:        "RebalanceStatus",
+			Method:      "GET",
+			Pattern:     "/volumes/{volname}/rebalance",
+			Version:     1,
```

Comment:
Please update other fields of route.Route structure. This is required for [API doc](https://github.com/gluster/glusterd2/blob/master/doc/endpoints.md) generation.
[Here](https://github.com/gluster/glusterd2/blob/master/glusterd2/commands/volumes/commands.go#L22-L23) is an example.